recipes/hover/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "hover" %}
+{% set version = "0.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hover-{{ version }}.tar.gz
+  sha256: 2f84d974b13cbe2ae857782e1a18f2f40404ad750df87edde08cb99e3a44d9e0
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```

Comment:
The setup.py upstream has a lot of version requirements for the dependencies. Is there a reason why these are not reproduced here?

Reply:
The CI fails with

```
hover 0.5.0 requires wrappy, which is not installed.
hover 0.5.0 has requirement snorkel>=0.9.3, but you have snorkel 0.7.0b0.
```

That might be related.

Reply:
The setup.py upstream was not loose enough about the version requirements, which leads to dependencies conflicting in indirect ways in the conda-forge setting.

Reply:
Ok. But no version requirements at all seems a bit too loose then. (We could consider opening an upstream issue to fix the requirements if they are too strict.)

Reply:
Yes, that sounds good. Thanks!